src/normal/normal.jl

```diff
@@ -32,15 +32,26 @@ std(N::Normal) = sqrt.(var(N))
 
 residual(N::Normal, x) = cholesky(N.Σ).L \ (x .- N.μ)
 
-logpdf(N::Normal{T,U,V}, x) where {T<:Real,U,V} =
-    -dim(N) / 2 * log(2 * π) - 1 / 2 * logdet(N.Σ) - norm_sqr(residual(N, x)) / 2
-logpdf(N::Normal{T,U,V}, x) where {T<:Complex,U,V} =
-    -dim(N) * log(π) - logdet(N.Σ) - norm_sqr(residual(N, x))
+_nscale(T::Type{<:Real}) = T(0.5)
+_nscale(T::Type{<:Complex}) = one(real(T))
 
-entropy(N::Normal{T,U,V}) where {T<:Real,U,V} =
-    dim(N) / 2.0 * (log(2.0 * π) + 1) + logdet(N.Σ) / 2.0
-entropy(N::Normal{T,U,V}) where {T<:Complex,U,V} = dim(N) * (log(π) + 1) + logdet(N.Σ)
+_piconst(T::Type{<:Real}) = T(2π)
+_piconst(T::Type{<:Complex}) = real(T)(π)
 
+logpdf(N::Normal, x) =
+    -_nscale(eltype(N)) * (logdet(_piconst(eltype(N)) * N.Σ) + norm_sqr(residual(N, x)))
```

Comment:
```suggestion
logpdf(N::Normal{T}, x) where {T} =
    -_nscale(T) * (logdet(_piconst(T) * N.Σ) + norm_sqr(residual(N, x)))
```
JuliaFormatter might complain though so you'd need to fix that afterwards.